connector_dicom/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,69 @@
+=================
+Connector - Dicom
+=================
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fl10n--chile-lightgray.png?logo=github
+    :target: https://github.com/OCA/l10n-chile
+    :alt: OCA/l10n-chile
+
+|badge1| |badge2| |badge3|
+
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuracion
+=============
+
+*
+
+Uso
+===
+
+*
+
+Creditos
+========
+
+Contribuidores
+--------------
+
+* `Open Source Integrators <https://www.opensourceintegrators.com>`_:
+
+  * Maxime Chambreuil <mchambreuil@opensourceintegrators.com>
+
+* `Konos <https://www.konos.cl>`_:
+
+  * Nelson Ramírez Sánchez <nramirez@konos.cl>
+
+Mantenedores
+------------
+
+Ese modulo esta mantenido por Open Source Integrators.
```

Comment:
Keep only the maintainer names, remove everything else (it is already in the OCA template)